Packs/CreateHash/pack_metadata.json

```diff
@@ -2,7 +2,7 @@
     "name": "CreateHash",
     "description": "Simple wrapper of the https://docs.python.org/3/library/hashlib.html to produce sha512, sha256, sha1, md5 and Blake (simple hash) of a given text input",
     "support": "community",
-    "currentVersion": "1.0.2",
+    "currentVersion": "1.0.3",
     "author": "Joerg Stephan",
```

Comment:
Where is the deprecating notes in the files? + hidden: true

Reply:
Added in later commit. all deprecating notes now exist for CreateHash.

Reply:
@samuelFain  I still don't see it in here. Where is the deprecated in the name and description in this file?

Reply:
Fixed now :)